lib/model/frames/form.js

```diff
@@ -140,13 +142,14 @@ Form.Partial = class extends Form {};
 
 Form.Extended = class extends Frame.define(
   'submissions',        readable,       'lastSubmission', readable,
-  'excelContentType',   readable
+  'excelContentType',   readable,       'name',           readable
 ) {
   forApi() {
     return {
       submissions: this.submissions || 0,
       lastSubmission: this.lastSubmission,
-      excelContentType: this.excelContentType
+      excelContentType: this.excelContentType,
+      name: this.name,
```

Comment:
why is this necessary? does it not come through the def in some cases?

Reply:
you're right, this isn't necessary... something i tried along the way, i think